sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs

```diff
@@ -42,18 +42,16 @@ internal class PartitionPump
         private string ConsumerGroup { get; }
 
         /// <summary>
-        ///   The identifier of the Event Hub partition this partition pump is associated with.  Events will be
-        ///   read only from this partition.
+        ///   The context of the Event Hub partition this partition pump is associated with.
         /// </summary>
         ///
-        private string PartitionId { get; }
+        private PartitionContext AssociatedPartitionContext { get; }
```

Comment:
nit:  Does `Associated` buy us anything here?   Are we taking a second instance somewhere that I overlooked where we could be confused by the scope?

Reply:
No. Just didn't want to keep the `PartitionContext`. Does `Context` seem reasonable?

Reply:
I like `Context`  👍